packages/@react-native-windows/cli/src/index.ts

```diff
@@ -122,7 +122,7 @@ export async function generateWindows(
         cliVersion = rnwCliPkgJson.version;
       } catch {}
       const optScrubbed = scrubOptions(options);
-      Telemetry.client?.trackEvent({
+      Telemetry.client.trackEvent({
```

Comment:
Optional chaining was automatically removed here since we already tested `Telemetry.client` against null.